cluster/util/environment.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"os"
+	"strings"
+)
+
+func EnvironmentVariablesToMap() map[string]string {
+	m := make(map[string]string)
```

Comment:
`make(map[string]string,len(os.Environ()))` would be a bit better here (pre-allocated `map`).